state/execution.go

```diff
@@ -234,10 +234,10 @@ func (blockExec *BlockExecutor) Commit(
 	// ResponseCommit has no error code - just data
 
 	blockExec.logger.Info(
-		"Committed state",
+		"committed state",
 		"height", block.Height,
-		"txs", len(block.Txs),
-		"appHash", res.Data,
+		"num_txs", len(block.Txs),
+		"app_hash", fmt.Sprintf("%X", res.Data),
```

Comment:
@melekes I see PR https://github.com/tendermint/tendermint/pull/5960 which states that this should be logged as hex without the need to format, right? Is that feature currently live in Stargate?

Reply:
> Is that feature currently live in Stargate?

I think so.

> ��\x1c1�U��ȺXu}���D�\x1c\v߃#OY���8\x13\x19.

weird...

Reply:
This works regardless of the underlying logger, right (i.e. go-kit vs zerolog, etc...)? In other words, we apply this logic before the underlying logger API gets called, correct?

Reply:
> This works regardless of the underlying logger, right (i.e. go-kit vs zerolog, etc...)? In other words, we apply this logic before the underlying logger API gets called, correct?

correct. here is the code: https://github.com/tendermint/tendermint/blob/93719c2eb34670b7ae541384ec1b86c08fb327de/libs/log/tmfmt_logger.go#L86-L89

Reply:
maybe it's because AppHash has a type `tmbytes.HexBytes`, not `[]byte`

Reply:
Ahh, that explains it then. The SDK does not use `NewTMFmtLogger`. It uses a zerolog logger directly. I think we should keep this change then.